### PR TITLE
Fixes #620: scroll problem

### LINKED ISSUE #620: Doesn't scroll listview
Simply add some list items, enough for scrolling.
Doesn't scroll with touch input. 
Mousewheel <b>only.</b>
<b>Example</b>: http://jsfiddle.net/RZFU7/
<b>Browser:</b>
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36

And iOS too

### PR DESCRIPTION
**Pull request by watilde**
Fixes #620 scroll problem with touch input.

### CODE REVIEW

File: sass/base.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 // We fix position the body and scroll `.content`.
```
Comment (XhmikosR) [inline sass/base.scss]: BTW this comment no longer applies.

**Comment by XhmikosR** [comment]
@watilde: fetch and rebase please?

**Comment by watilde** [comment]
@XhmikosR Thanks for pick this PR! I've done rebase.

**Comment by XhmikosR** [comment]
@watilde: can you remove the build commit and squash the second.

**Comment by watilde** [comment]
@XhmikosR Sure. I've done too.

**Comment by XhmikosR** [comment]
@watilde: you didn't skip the dist files so the branch will conflict again if not merged soon. Just remove those.

**Comment by watilde** [comment]
@XhmikosR Oh I see for now that the branch will conflict again if I comitted dist files. I just removed that and pushed again!

**Comment by watilde** [comment]
@XhmikosR I just rebased even this patch is created 2 years ago :)

**Comment by XhmikosR** [comment]
Sorry, I've moved to other things myself since Ratchet seems to be abandoned :/

**Comment by watilde** [comment]
Yeah, it seems so, unfortunately. I even think Ratchet is still unique and useful at least, I got it. I will just leave this PR, and hope it will be merged someday.

Thanks anyway! I was a very beginner of git when I created this patch xD

**Comment by watilde** [comment]
Closing because of the current status of this project.

**Pull request closed without merging**